app/controllers/rhapsody_realities.js

```diff
@@ -39,6 +39,18 @@ class RhapsodyController {
 
         res.render('admin_add_rhapsody', {layout : 'admin_main'});
     }
+
+     /**
+     * rhapsody create
```

Comment:
comment here says create why functionality is actually list 🤔

Reply:
Ok i get now.